### PR TITLE
adding a bunch of logging

### PR DESCRIPTION
**Pull request by mhines-usgs**
trying to figure out why template creation is failing

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 1]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
...
 		String summaryJson = null;
+		log.debug("summaryJson", summaryJson);
 		try {
 			summaryJson = MetadataUtil.getSummaryFromWPS(cswEndpoint, attr);
+			log.debug("summaryJsonMetadataUtil", summaryJson);
 		} catch (IOException | ParserConfigurationException | SAXException | URISyntaxException ex) {
 			log.error("Problem getting summary from item", ex);
 		}
 		Gson gson = GsonUtil.getDefault();
+		log.debug("gson", gson);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: This won't exactly work how you want, gson is an object and it doesn't know how to log in this case.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 2]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
...
 		Summary summary = gson.fromJson(summaryJson, Summary.class);
+		log.debug("summary", summary);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Same with summary, the logger doesn't know what to do here.

**Comment by mhines-usgs** [comment]
@jiwalker-usgs not sure if this makes a difference here with these two objects (using String.valueOf()) -- if you think it'll work, or won't let me know.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 3]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
 
 	private Summary makeSummary(Layer layer, String attr) throws JsonSyntaxException {
 		OGCService cswService = Service.ogcHelper(Service.ServiceType.csw, layer.getServices());
+		log.debug("cswService", cswService);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Switch this to `log.debug("cswService {}", cswService)`

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 4]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
...
 		String cswEndpoint = cswService.getEndpoint();
+		log.debug("cswEndpoint", cswEndpoint);
 		String summaryJson = null;
+		log.debug("summaryJson", summaryJson);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: Drop this line.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 5]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
...
 		try {
 			summaryJson = MetadataUtil.getSummaryFromWPS(cswEndpoint, attr);
+			log.debug("summaryJsonMetadataUtil", summaryJson);
 		} catch (IOException | ParserConfigurationException | SAXException | URISyntaxException ex) {
 			log.error("Problem getting summary from item", ex);
 		}
 		Gson gson = GsonUtil.getDefault();
+		log.debug(String.valueOf(gson));
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: These are fine.

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 6]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
...
 	private Summary makeSummary(Layer layer, String attr) throws JsonSyntaxException {
 		OGCService cswService = Service.ogcHelper(Service.ServiceType.csw, layer.getServices());
+		log.debug("cswService", cswService);
 		String cswEndpoint = cswService.getEndpoint();
+		log.debug("cswEndpoint", cswEndpoint);
 		String summaryJson = null;
+		log.debug("summaryJson", summaryJson);
 		try {
 			summaryJson = MetadataUtil.getSummaryFromWPS(cswEndpoint, attr);
+			log.debug("summaryJsonMetadataUtil", summaryJson);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: also add the {}

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java [hunk 7]
```diff
@@ -252,15 +252,21 @@ private Item templateItem(Item template, String attr, Layer layer, Summary summa
 
 	private Summary makeSummary(Layer layer, String attr) throws JsonSyntaxException {
 		OGCService cswService = Service.ogcHelper(Service.ServiceType.csw, layer.getServices());
+		log.debug("cswService", cswService);
 		String cswEndpoint = cswService.getEndpoint();
+		log.debug("cswEndpoint", cswEndpoint);
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/rest/data/TemplateResource.java]: last {}

**Pull request merged**